vcbuild.bat

```diff
@@ -117,7 +117,7 @@ if "%i18n_arg%"=="full-icu" set configure_flags=%configure_flags% --with-intl=fu
 if "%i18n_arg%"=="small-icu" set configure_flags=%configure_flags% --with-intl=small-icu
 if "%i18n_arg%"=="intl-none" set configure_flags=%configure_flags% --with-intl=none
 if "%i18n_arg%"=="without-intl" set configure_flags=%configure_flags% --without-intl
-if "%engine%"=="chakracore" set configure_flags=%configure_flags% --without-intl --without-inspector
+if "%engine%"=="chakracore" set configure_flags=%configure_flags% --without-intl --without-inspector --without-v8-platform --without-bundled-v8
```

Comment:
yes. Without these options, `node.gyp` will pick the default values which is true for all of them. To not have specify in `vcbuild.bat`, we will have to put everything that these switches corresponds to under `engine=v8` inside `node.gyp` which i feel is too much of change. Having commands here is simple with minimal changes.